src/de.rs

```diff
@@ -15,13 +15,16 @@ use crate::error::{Error, ErrorCode, Result};
 use crate::read::EitherLifetime;
 #[cfg(feature = "unsealed_read_write")]
 pub use crate::read::EitherLifetime;
-use crate::read::Offset;
 #[cfg(feature = "std")]
-pub use crate::read::{IoRead, SliceRead};
+pub use crate::read::IoRead;
+use crate::read::Offset;
+#[cfg(any(feature = "std", feature = "alloc"))]
+pub use crate::read::SliceRead;
 pub use crate::read::{MutSliceRead, Read, SliceReadFixed};
-
 /// Decodes a value from CBOR data in a slice.
 ///
+/// Only available with the `std` (default) or `alloc` feature.
```

Comment:
Bit of a nit: I think this could be omitted, as presumably 99% of users will just use the `std` version. The remaining 1% that do care about `no_std` can build the documentation without the `std` feature to see that this function is unavailable (or browse the source).

Reply:
Makes sense. I remove these comments.